WebCryptoAPI/util/helpers.js

```diff
@@ -232,6 +235,6 @@ function allNameVariants(name, slowTest) {
 
     // for slow tests effectively cut the amount of work in third by only
     // returning one variation
-    if (slowTest) return [mixedCaseName];
-    return [upCaseName, lowCaseName, mixedCaseName];
+    if (slowTest) return unique([mixedCaseName]);
```

Comment:
Thanks! This is a bit nitpicky but I think this one isn't really necessary :)

Reply:
out of all the things in this suite ;)